### ISSUE TITLE
Mac OS X - pip install tables "Could not find a local HDF5 installation"

### ISSUE THREAD

**Issue by seanfuture** [PyTables/PyTables#676]
When installing tables received the error message "Could not find a local HDF5 installation".  Packages should install without error regardless of dependencies.  If your dependencies are faulty and or cannot be installed via pip you should choose different dependencies and/or enhance them so that they can be installed via pip.

Full error log:
```
 ~ > pip install tables
Collecting tables
  Using cached tables-3.4.2.tar.gz
    Complete output from command python setup.py egg_info:
    /var/folders/m6/v3skzt510wqblhwh3t8m9spc0000gn/T/H5closeqc6h71df.c:1:1: warning: type specifier missing, defaults to 'int' [-Wimplicit-int]
    main (int argc, char **argv) {
    ^
    /var/folders/m6/v3skzt510wqblhwh3t8m9spc0000gn/T/H5closeqc6h71df.c:2:5: warning: implicit declaration of function 'H5close' is invalid in C99 [-Wimplicit-function-declaration]
        H5close();
        ^
    2 warnings generated.
    ld: library not found for -lhdf5
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    * Using Python 3.4.1 (default, May 19 2014, 13:10:29)
    * USE_PKGCONFIG: True
    .. ERROR:: Could not find a local HDF5 installation.
       You may need to explicitly state where your local HDF5 headers and
       library can be found by setting the ``HDF5_DIR`` environment
       variable or by using the ``--hdf5`` command-line option.
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/m6/v3skzt510wqblhwh3t8m9spc0000gn/T/pip-build-98w_39dh/tables/

```

**Comment by tomkooij** [comment]
`pip` is installing `tables` from source, because there are no pre-built wheels available for python-3.4 on Mac OSX.

If you can upgrade to python>=3.5 you can you the wheels that come with HDF5. If you need to stay on python-3.4 you need to install the HDF5 libraries as described in the installation sections of the user's guide: http://www.pytables.org/usersguide/installation.html

**Comment by avalentino** [comment]
+1
